src/Common/src/Internal/Cryptography/CryptoThrowHelper.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Security.Cryptography;
+
+namespace Internal.Cryptography
+{
+    internal static class CryptoThrowHelper
```

Comment:
Internal\Cryptography\Windows is OK with me, as is the namespace mis-match (in X509 the PAL is Pal.Unix and Pal.Windows; but the namespace is just "Pal").

Or, if you wanted to put it in Interop\Windows\BCrypt that would make some sense (the CSP library forward-reaching for the errors helper seems fine to me)